dot/network/test_helpers.go

```diff
@@ -190,12 +194,16 @@ var starting, _ = variadic.NewUint64OrHash(uint64(1))
 
 var one = uint32(1)
 
-var testBlockRequestMessage = &BlockRequestMessage{
-	RequestedData: RequestedDataHeader + RequestedDataBody + RequestedDataJustification,
-	StartingBlock: *starting,
-	EndBlockHash:  &common.Hash{},
-	Direction:     1,
-	Max:           &one,
+func NewTestBlockRequestMessage(t *testing.T) *BlockRequestMessage {
```

Comment:
Do we need to export this? Also, if not, can you rename `test_helpers.go` to `helpers_test.go` so it's treated as a test file?

Reply:
file and function renamed!